### ISSUE TITLE
Manifest.json key error if source maps = true and extensionless = input

### ISSUE THREAD

**Issue by woodcox** [jfortunato/esbuild-plugin-manifest#20]
Hi, the error you fixed with the 1.0.1 version, has resulted in another error when source maps are set to true in the esbuild config and the manifest plugin is configured with `extensionless: 'input'`.

For example:

```js
const esbuildOpts = {
  sourcemap: true,
  // other config...
  plugins:[ 
    manifestPlugin({
      hash: isProd ? '[name]-[hash]' : '[name]',
      shortNames: true,
      extensionless: 'input'
    }),
  // more config ...
  ]
}
```

This config results in an error message in the terminal: 

[ERROR] There is a conflicting manifest key for 'app'. First conflicting output: 'app.js.map'. Second conflicting output: 'app.js'. [plugin manifest].

I believe this is because there cannot be two `app` keys for `app.js.map` and the `app.js` in the `manifest.json`. To resolve this you could append `.map` to the key when the value ends in .map?

**Comment by jfortunato** [comment]
Should be fixed with `v1.0.2`

**Comment by woodcox** [comment]
Great, thanks again for updating so quickly. That's fixed the issue.
